classy/gui.py

```diff
@@ -5,10 +5,14 @@
 import classy
 from classy.log import logger
 
-import pyqtgraph as pg
-
-# from pyqtgraph.Qt import QtCore, QtWidgets
-from pyqtgraph.Qt import QtWidgets, QtGui
+try:
+    import pyqtgraph as pg
+    from pyqtgraph.Qt import QtWidgets
+except ModuleNotFoundError as e:
+    raise ModuleNotFoundError(
+        "The Graphical User Interface of classy requires the pyqt5 package. "
```

Comment:
I wonder if this is used/necessary since pyqtgraph already includes a more specific error message about all the packages it could use, though unfortunately as a generic Exception:
`Exception: PyQtGraph requires one of PyQt5, PyQt6, PySide2 or PySide6; none of these packages could be imported`

Reply:
Agree that it's not necessary but I want to give the solution to copy/paste to the user, make it really simple for them.

Reply:
```suggestion
    raise ModuleNotFoundError(
        "The Graphical User Interface of classy requires the pyqtgraph and pyside6 packages. "
```